README.md

```diff
@@ -46,7 +69,77 @@ npm install
 
 ## Usage
 
-TBD
+This client assumes:
+
+* a remote Wallet Attached Storage server is available
+* the app is able to create and store a public-private key pair, see the
+  [App Identity and Key Management](#app-identity-and-key-management) section
+  for more details.
+
+Example key pair generation (for an in-browser SPA), for persistence in local
+storage.
+
+```ts
+import { Ed25519Signer } from '@did.coop/did-key-ed25519'
+
+const appDidSigner = await Ed25519Signer.generate()
+```
+
+Example storing and loading:
+
+```ts
+// Serialize to JSON for storage
+const exportedKeyPair = appDidSigner.toJSON()
+localStorage.setItem('app-DID', JSON.stringify(exportedKeyPair))
```

Comment:
Hmm. Ideally these examples would store non exportable secret keys.

Unfortunately, that is only possible for ed25519 keys on safari and firefox atm. chromium 🔜  https://github.com/WICG/webcrypto-secure-curves/issues/20

https://caniuse.com/mdn-api_subtlecrypto_sign_ed25519

I don't mind leaving this for now, but I'm curious what you think of:
* we should support secp256r1 keys/sigs not only ed25519. iiuc these are FIPS approved and supported across browsers via WebCrypto
* then we can maybe have some kind of generic WebCrypto signer that uses the most secure curve supported by the user agent, all non-exportable

Reply:
Totally agree, re non-exportable keys. Unfortunately we're just supporting ed25519 for the first iteration, so I figured for the first spike, we can do localstorage.